### PR TITLE
Fix languages test install

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | The languages installed for tests fail randomly, sometimes only English is present sometimes french and English This PR aims to uniform this behaviour to have a consistent test environment
| Type?             | bug fix
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| How to test?      | ~
| Possible impacts? | ~

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25320)

### CODE REVIEW

File: src/PrestaShopBundle/Install/DatabaseDump.php [hunk 1]
```diff
@@ -57,7 +58,7 @@ private function __construct($dumpFile = null)
         }
 
         if ($dumpFile === null) {
-            $this->dumpFile = sys_get_temp_dir() . '/' . 'ps_dump.sql';
+            $this->dumpFile = sprintf('%s/ps_dump_%s.sql', sys_get_temp_dir(), AppKernel::VERSION);
```
Comment (PierreRambaud) [inline src/PrestaShopBundle/Install/DatabaseDump.php:61]: You have to also change DatabaseCreator.php about the ps_dump.sql

**Comment by matks** [comment]
@jolelievre can we close this PR?

**Comment by jolelievre** [comment]
I wonder, this one still brings a little bit of value We don't need to install the shop initially since Databasecreator can now handle it all

**Review by atomiix — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by atomiix** [comment]
Thanks @jolelievre!

**Pull request merged**